server/controllers/clubProfileController.js

```diff
@@ -1,6 +1,54 @@
-const clubOverview = require("../models/ClubOverviewModel")
 const clubProfile = require("../models/ClubProfileModel")
 
+const findSimilarClubs = (res, clubResult) => {
+    clubProfile.aggregate([
```

Comment:
for clarification, is this aggregate function going to return a list of db objects, or only their ids?